### DISCUSSION TITLE
请教关于递归的问题和外网应用静态IP的问题请

### DISCUSSION THREAD

**Discussion by Makerlong** [kkkgo/PaoPaoDNS#71] [问答和帮助] [answered]
## 环境
ikuai+paodns*2+paogateway
## 问题一：递归相关
我目前在windows执行下面代码:
```
nslookup whoami.03k.org 192.168.8.3
```
返回的信息如下：
```
20231123115017.710.reject.paopaodns     text =

        "0ms, END by try_auto_check."
服务器:  UnKnown
Address:  192.168.8.3

非权威应答:
名称:    whoami.03k.org
Address:  120.245.140.140
```
其中120.245.140.140这个IP是我之前的公网地址，而我现在的公网IP地址是其他的地址。
请教一下是需要刷新一下paodns的缓存么，还是清理一下电脑的dns缓存，还是这种情况是正常的。

## 问题二：外网应用静态路由的问题
目前我外网是正常的访问，但是流媒体迪士尼，我查迪士尼可能是域名的吧是无法正常观看的（节点是支持的），这个问题也是要和电报、奈非一样增加静态IP么。如果需要增加这些静态IP是在dns的哪个文件中增加呢，还是在ikuai的那个静态路由里面增加呢。

### 以上是所有问题，麻烦大佬帮忙解答下，谢谢。

**Comment by kkkgo** [comment]
1、是有缓存，或者服务器没响应，正常，你可以多nslookup几次或者用akahelp的服务器测试`nslookup -type=TXT whoami.ds.akahelp.net`。

2、参考https://github.com/kkkgo/PaoPaoGateWay/discussions/32#discussioncomment-7447476

**Reply by Makerlong** [reply]
迪士尼的话我没找到IP好像是域名的（我也不知道emm），我目前没法访问emmm。netfilx我找到了。
静态路由IP的话加在ikuai上就可以了么，是我下面这个图这个么设置么。
![image](https://github.com/kkkgo/PaoPaoDNS/assets/129925998/b4b23690-35a2-49f8-aece-206e5fb199e3)

**Reply by kkkgo** [reply, marked as answer]
是这样设置，不过爱快这种系统按理说应该可以分组添加的吧，不用一条条加。
你先确认用电脑浏览器能不能访问排除是节点问题，电脑浏览器必定走域名连接。
其次如果是有IPv6关闭IPv6后测试，如果是电视盒子可以设置网关和DNS都为PPGW的来测试。

**Reply by Makerlong** [reply]
好的谢谢大佬
